ansible/roles/elastic-stack/vars/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+
+elastic_compose_project_name: elastic
```

Comment:
elk ?

Reply:
It doesn't make sense, the term is deprecated and we use filebeat instead of logstash. Container names are currently:
```
elastic_kibana_1
elastic_elasticsearch_1
```
It's ok?